.env.test

```diff
@@ -14,3 +14,6 @@ SITE_URL="test.host"
 
 OPENID_APP_ID="test-provider"
 OPENID_APP_SECRET="12345"
+
+CLIENT_SECRET =~ /secret.+/
+HOSTNAME =~ /"hostname":".+"/
```

Comment:
I agree, both client_secret and data on hostname should not be public.

I've tried this approach to hide sensitive data, but it does not seem to work. I'm not sure the expressions are being evaluated correctly.

Any suggestions @dacook ?

Reply:
We can't use patterns here. And I couldn't find any use of patterns in the docs. We have to provide the real values to VCR while recording. We do that by fetching the secrets from the environment and tell VCR to filter those values.

So we would need to find out how the hostname is computed and do the same. And where does the client secret come from?

Reply:
Fixed it ! see https://github.com/openfoodfoundation/openfoodnetwork/pull/12252/commits/a33eb80f56c4fc80c0d38086982019c6b8bb4b69